infra/cifuzz/build_fuzzers.py

```diff
@@ -95,8 +95,8 @@ def build_fuzzers(self):
     ])
     rm_path = os.path.join(self.image_repo_path, '*')
     image_src_path = os.path.dirname(self.image_repo_path)
-    bash_command = (f'rm -rf {rm_path} && cp -r {self.host_repo_path} '
-                    f'{image_src_path} && compile')
+    bash_command = (f'cd / && rm -rf {rm_path} && cp -r {self.host_repo_path} '
```

Comment:
This change is to avoid some warnings from bash that can result from rming the directory we are currently in.